millipede/__init__.py

```diff
@@ -35,6 +38,34 @@ def millipede(size, comment=None, reverse=False):
 
     return output
 
+
+def send_sms(message, phone_number, api_login, api_passwd):
+    """ Send `message` to `phone_number` using the Rentabiliweb SMS API.
+    It requires to have a contract with Rentabiliweb, valid credentials and to
+    send the message from a whitelisted IP address.
+    """
+    try:
+        import requests
+    except ImportError:
+        print('requests is required to send SMS.', file=sys.stderr)
+        sys.exit(1)
+
+    api_url = 'https://sms.v3.eiole.com/send'
```

Comment:
I think this value should be placed in a configuration management system.

What about etcd?